### ISSUE TITLE
Add instructions for manual installation

### ISSUE THREAD

**Issue by pekkaklarck** [robotframework/SeleniumLibrary#255]
`pip install robotframework-selenium2library` is awesome but unfortunately impossible w/o network access and problematic with some proxy configurations. The installation instructions should contain steps how to install the library and its dependencies manually.

If I remember correctly, not all dependencies have Windows installers, and thus everyone must install at least some package using the old-school `python setup.py install` style. It is thus easiest to explain installing all the dependencies, and possibly also the library itself, using that approach.

**Comment by pekkaklarck** [comment]
I wrote better installation instructions to README.rst and submitted them as pull request #261. As I wrote in the commit message to 1fb0158, I noticed that this project also has more detailed instructions in INSTALL.rst only after writing the new instructions. That file was just mentioned in the beginning of README.rst, not under its Installation section. Information in INSTALL.rst was also partly outdated.

Having detailed instructions both in README.rst and in INSTALL.rst obviously makes no sense. I think there are two possibilities:

1) Remove separate INSTALL.rst. There seems to be some good stuff we could move to the new instructions before removing it, though.

2) Move new instructions for INSTALL.rst, keeping the good parts of both. Remove detailed instructions from README.rst, but make link to INSTALL.rst more visible.

Any comments which approach to use? I can continue working with either and send a new pull request when done.

**Comment by pekkaklarck** [comment]
After thinking this a bit more, I would say it's better to keep installation instructions in README.rst and remove INSTALL.rst altogether. These instructions are anyway fairly short, and it would be convenient for users to have everything in one place both in GitHub front page and, if #254 is accepted, also in PyPI. It would also be easier to maintain just single file instead of two files.

Unless others disagree, I'll continue enhancing the instructions by removing INSTALL.rst and updating README.rst accordingly. Before removing the file, I'll see if there's anything worth keeping that should be moved to README.rst.

**Comment by aaltat** [comment]
Do we still need this? We are writing better documentation for the library but is the manual installation really needed?

**Comment by pekkaklarck** [comment]
No.
